cartography/intel/okta/applications.py

```diff
@@ -1,47 +1,46 @@
 # Okta intel module - Applications
 import json
 import logging
+import socket
```

Comment:
What socket functionality is used? Is this something we can do with requests like the other modules?

Reply:
socket is used to check the dns for the domain of each reply url.